fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -23,7 +23,7 @@ module OCamlInterop = LibBackend.OCamlInterop
 module G = Generators
 
 /// Ensure we only work with OCaml-friendly floats
-let filterFloat (f : float) : bool =
+let isValidOCamlFloat (f : float) : bool =
```

Comment:
I suspect this can be removed. I think it was about passing them to ocaml via json, and I'm pretty sure that's supported now.

Reply:
Fyi these are all "valid" floats (floats are floats whereever they are)

Reply:
Especially since `NaN <> NaN`